<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>630</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BGR</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>830</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ADJUST MODE</string>
    </property>
   </widget>
   <widget class="QSlider" name="sdBlue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>790</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sdGreen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>760</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sdRed">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>730</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>730</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelG">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>760</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>790</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPencil">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pencil</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbRed">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>730</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbGreen">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>760</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbBlue">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>790</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnErase">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStore">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Store</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPencil_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Adjust Mode</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
